<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>userinfoDialog</class>
 <widget class="QDialog" name="userinfoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>528</width>
    <height>425</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
background:#DCDCDC;
border-right:1px solid black;
border-left:1px solid black;
border-top:1px solid black;
border-bottom:1px solid black;
border-top-left-radius:10px;
border-bottom-left-radius:10px;
border-top-right-radius:10px;
border-bottom-right-radius:10px;
}
QPushButton{
    border:none;
    border-bottom:1px solid black;
    font-size:18px;
    font-weight:700;
 }
QPushButton:hover{border-left:4px solid grey;font-weight:700;}
QCheckBox{
border:none}
QLabel{border:none}</string>
  </property>
  <widget class="QLabel" name="UserInfo">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>30</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文琥珀</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>个人信息</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>310</y>
     <width>93</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QLabel" name="msgLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>380</y>
     <width>281</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="usernumberLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>100</y>
     <width>72</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文中宋</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>学号</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="UsernumlineEdit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>100</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>请输入10位学号</string>
   </property>
  </widget>
  <widget class="QLabel" name="usernameLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>170</y>
     <width>72</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文中宋</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>姓名</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="UsernamelineEdit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>170</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>请输入10位内汉字</string>
   </property>
  </widget>
  <widget class="QLabel" name="EnglishLabel">
   <property name="geometry">
    <rect>
     <x>81</x>
     <y>240</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文中宋</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>英文名</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="EnglishnamelineEdit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>240</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>请输入10个字母以内</string>
   </property>
  </widget>
  <widget class="QPushButton" name="left_close">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>10</y>
     <width>31</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="left_visit">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>10</y>
     <width>31</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="left_mini">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>10</y>
     <width>31</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
